pom.xml

```diff
@@ -138,6 +142,12 @@
         </dependency>
 
         <dependency>
+            <groupId>javax.mail</groupId>
```

Comment:
Just to confirm, I ran the application on master branch now. The exception is not seen.